### ISSUE TITLE
Scroll up to zoom in does not work, but zoom out does

### ISSUE THREAD

**Issue by Sheresik** [linearmouse/linearmouse#861]
### OS

macOS 15.3 Beta (24D5034f)

### LinearMouse

v0.10.0

### Describe the bug

Not sure if it is a bug, or I'm doing something wrong. I have set the Modifier Keys (Control) to Zoom hoping to use it as windows ctrl+scroll, but so far I managed to only zoom out, scrolling to front does not work at all. 

### To reproduce

Set a Midifier key (any) to "Zoom" and try to scroll with a mouse button

### Expected behavior

Zoom in

**Comment by lujjjh** [comment]
Thanks for your feedback!

Does this happen in all apps or only in a specific one? Additionally, could you let me know what keyboard layout you are using?

**Comment by Sheresik** [comment]
I was just testing everything to write it down to you and found the issue... 
I'm using Czech Keyboard (QWERTZ) version, which I believe is causing the issue. Changing the keyboard layout to regular English ANSI keyboard solves it, but I barely ever use regular English keyboard. 

Do you think it could be fixed in the future updates for Czech QWERTZ version?
Thanks and sorry!

**Comment by lcq92** [comment]
I have the same issue on Macbook! Here is what I observe:

- Using the Swiss-German or Swiss-French QWERTZ layouts, I find that on _Chrome_ and _TextEdit_, only scrolling down to zoom out works, but the opposite doesn't work. However, on _Sublime Text_ and _VSCode_, both directions work well

- When switching to US (QWERTY) layout, both directions work well on all four apps mentioned above.

**Comment by lujjjh** [comment]
It looks like the zoom-in shortcut on the Swiss-French QWERTZ layout is Command + ! instead of Command + +. Not sure if this applies to all apps?

**Comment by Sheresik** [comment]
You are right! Here it is actually command + % on Safari at least, I never thought that different layouts for something so obvious like + and -.

**Comment by nandorholozsnyak** [comment]
Hello there,
It happens with Hungarian Keyboard and layout (QWERTZ) too.

**Comment by Firu115** [comment]
Same here. Czech QWERTZ.

**Comment by nemethbalintkristof-design** [comment]
I think I managed to solve this issue. I had the same problem mentioned above: zooming out worked, but zooming in did not. My keyboard layout is Hungarian QWERTZ.

On macOS, Command + – is used to zoom out, while Command + + is used to zoom in. Zooming out worked on my keyboard, but zooming in did not. To fix this, I went to System Settings → Keyboard → Keyboard Shortcuts → App Shortcuts, and added a new shortcut with the menu title “Zoom In” and the shortcut Command + +.

This solved the zoom-in issue for me when using Ctrl + scrolling. The only remaining problem is that I can’t zoom in using the keyboard alone, because my Mac keyboard doesn’t have a dedicated + key (only the external keyboard has). I can only type + using Shift + 3, but that combination doesn’t work for zooming in with the Command key.

Hope this helps.

<img width="737" height="654" alt="Image" src="https://github.com/user-attachments/assets/a14e4c85-12ae-48d5-87c0-fb4097ad77b3" />

**Comment by ertugruluyar-github** [comment]
Same with tr-TR Q keyboard.
